### DISCUSSION TITLE
MR description keeps changing

### DISCUSSION THREAD

**Discussion by Turbo87** [renovatebot/renovate#17116] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

32.105.0

### Please select which platform you are using if self-hosting.

GitLab self-hosted

### If you're self-hosting Renovate, tell us what version of the platform you run.

GitLab Enterprise Edition 14.10.2-ee

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

Renovate keeps adjusting the MR body for some reason:

<img width="843" alt="Bildschirmfoto 2022-07-05 um 09 43 30" src="https://user-images.githubusercontent.com/141300/177277280-5edb10b6-7227-4c10-adb0-5672259cf9d6.png">

<img width="1043" alt="Bildschirmfoto 2022-07-05 um 09 42 22" src="https://user-images.githubusercontent.com/141300/177277292-e143509b-b972-4a1b-b514-506a538e8fb2.png">

We run renovate on an hourly schedule and the MR body keeps flipping between the two states above.

### Relevant debug logs

<details><summary>Logs</summary>

```
{"name":"renovate","hostname":"runner-xhw4dwks-project-298-concurrent-0","pid":173,"level":20,"logContext":"1RBgiuxTNjvsgHf8Z3h3R","repository":"npm-packages/ember-template-lint-plugin-qonto","branch":"deps-db0e148468","msg":"getPr(133)","time":"2022-07-05T07:03:45.282Z","v":0}
{"name":"renovate","hostname":"runner-xhw4dwks-project-298-concurrent-0","pid":173,"level":20,"logContext":"1RBgiuxTNjvsgHf8Z3h3R","repository":"npm-packages/ember-template-lint-plugin-qonto","branch":"deps-db0e148468","msg":"getMR(133)","time":"2022-07-05T07:03:45.282Z","v":0}
{"name":"renovate","hostname":"runner-xhw4dwks-project-298-concurrent-0","pid":173,"level":20,"logContext":"1RBgiuxTNjvsgHf8Z3h3R","repository":"npm-packages/ember-template-lint-plugin-qonto","branch":"deps-db0e148468","msg":"Found existing PR","time":"2022-07-05T07:03:45.284Z","v":0}
{"name":"renovate","hostname":"runner-xhw4dwks-project-298-concurrent-0","pid":173,"level":20,"logContext":"1RBgiuxTNjvsgHf8Z3h3R","repository":"npm-packages/ember-template-lint-plugin-qonto","branch":"deps-db0e148468","msg":"Processing existing PR","time":"2022-07-05T07:03:45.302Z","v":0}
{"name":"renovate","hostname":"runner-xhw4dwks-project-298-concurrent-0","pid":173,"level":20,"logContext":"1RBgiuxTNjvsgHf8Z3h3R","repository":"npm-packages/ember-template-lint-plugin-qonto","branch":"deps-db0e148468","prTitle":"Update dependency ember-template-lint to v4.10.1","msg":"PR body changed","time":"2022-07-05T07:03:45.303Z","v":0}
{"name":"renovate","hostname":"runner-xhw4dwks-project-298-concurrent-0","pid":173,"level":30,"logContext":"1RBgiuxTNjvsgHf8Z3h3R","repository":"npm-packages/ember-template-lint-plugin-qonto","branch":"deps-db0e148468","pr":133,"prTitle":"Update dependency ember-template-lint to v4.10.1","msg":"PR updated","time":"2022-07-05T07:03:46.165Z","v":0}
{"name":"renovate","hostname":"runner-xhw4dwks-project-298-concurrent-0","pid":173,"level":20,"logContext":"1RBgiuxTNjvsgHf8Z3h3R","repository":"npm-packages/ember-template-lint-plugin-qonto","branch":"deps-db0e148468","msg":"PR is not configured for automerge","time":"2022-07-05T07:03:46.165Z","v":0}
```

</details>

It only says "PR body changed", but unfortunately it does not appear to mention why it was changed.

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment]
Are you sure you don't have an older version of Renovate running at the same time?

**Comment by Turbo87** [comment]
@rarkins I was wondering the same thing, but couldn't find anything. given that it flip-flops every hour and not twice per hour I would assume it's the same version.

**Comment by rarkins** [comment]
I don't understand how it could flip between the old version and new though

**Comment by PhilipAbed** [comment]
can you please take the 2 PRs, and do a compare between them to show us the difference?
they both look exactly the same to me, we dont have enough information to decide whats the issue here

**Comment by Turbo87** [comment]
@PhilipAbed I'm not sure I understand what you're requesting. There is only one PR but the PR body is flip-flopping all the time (well, to be precise, this is happening on all GitLab MRs from renovatebot for us).

you can see the diff between the two states in the second screenshot in the issue description above.

**Comment by PhilipAbed** [comment]
i mean go to the Files Changed like in the screenshot, 
Take the one that inlcudes the RED and the one that includes the Green, and text compare them to see what is actually changing!

thing is i look at them and they look the same to me, so whats different? why is github deciding that they are different?

**Comment by rarkins** [comment]
One screenshot showed red and one green. Ie part of the sentence is being added and removed repeatedly.

**Comment by Turbo87** [comment, marked as answer]
> Are you sure you don't have an older version of Renovate running at the same time?

turns out you were right. devops team had a second staging gitlab instance running with an outdated clone of the original repo incl. its CI jobs and schedules 🙈 

sorry for the noise everyone 😅
